### DISCUSSION TITLE
v0.10.0-preview2

### DISCUSSION THREAD

**Discussion by Blinue** [Blinue/Magpie#484] [Announcements]
仍在征集新图标 #370

**运行环境变更**

不再需要 MSVC 运行时。现在无需用户安装任何环境。

**新功能**
1. 本地化支持，添加了英语本地化（*Added English localization*）
2. 自动更新 #287
3. 自动缩放 #193
4. 优化了对打包应用的支持
    * 打包应用指通过 appx 安装的应用，如 UWP
5. 标题栏菜单和系统托盘菜单适配深色主题
6. 很多性能优化

**修复**
1. 修复了以管理员身份运行时导入/导出闪退的问题
2. 现在始终以用户身份启动游戏，即使 Magpie 是管理员身份
3. 修复了重启资源管理器后托盘图标丢失的问题 #448
4. 优化了快捷键的可用性 #448
8. 修复了与 RTSS 的兼容性
9. 现在禁止缩放操作系统的特殊窗口，如开始菜单、任务栏、桌面窗口等
10. 现在管理员身份下不支持拖拽排序 #457

**效果变更**

添加 Deband #479 （感谢 @kato-megumi）

<hr /><em>This discussion was created from the release <a href='https://github.com/Blinue/Magpie/releases/tag/v0.10.0-preview2'>v0.10.0-preview2</a>.</em>

**Comment by Crachlow** [comment]
Thanks Blinue for the English.

**Comment by luojunyuan** [comment]
Is it possible to publish Magpie on micrsoft store in the future.

**Reply by Blinue** [reply]
This is in the plan! I will try to publish it on the Microsoft Store when version 1.0 is released. The next version after v0.10 will be v1.0

**Comment by Crachlow** [comment]
Blinue. What is 3D game mode?

**Reply by Blinue** [reply]
When enabled, the behavior of the cursor and in-game overlay will be changed to optimize for 3D games. It is recommended to turn on this option when scaling 3D games.

**Comment by Crachlow** [comment]
Blinue. Will it be possible in the future to implement the connection of other languages, in particular Russian.

**Reply by Blinue** [reply]
#491

**Reply by 0x4E69676874466F78** [reply]
> Will it be possible in the future to implement the connection of other languages, in particular Russian.

Русский язык уже есть в поставке. [Возьми свежую сборку из последних Actions.](https://github.com/Blinue/Magpie/actions/runs/4444856976)

**Reply by Crachlow** [reply]
> > Will it be possible in the future to implement the connection of other languages, in particular Russian.
> 
> Русский язык уже есть в поставке. [Возьми свежую сборку из последних Actions.](https://github.com/Blinue/Magpie/actions/runs/4444856976)

Blinue. Thank you. You are super.
